framework/src/node/generator/strategies.ts

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import { StateStore } from '@liskhq/lisk-chain/dist-node/state_store';
+import { TransactionPool, PooledTransaction } from '@liskhq/lisk-transaction-pool';
+import { dataStructures } from '@liskhq/lisk-utils';
+import { Chain, Transaction } from '@liskhq/lisk-chain';
+import { getAddressFromPublicKey } from '@liskhq/lisk-cryptography';
+import { KVStore } from '@liskhq/lisk-db';
+import { StateMachine, VerifyStatus, EventQueue, TransactionContext } from '../state_machine';
+import { Logger } from '../../logger';
+import { WritableBlockHeader } from './types';
+
+export class HighFeeGenerationStrategy {
```

Comment:
this is copied from `strategies` in `forger` fodler.
Updated to use state machine instead of processor